lib/src/chatbox.dart

```diff
@@ -310,23 +338,23 @@ class ChatBoxState extends State<ChatBox> {
 
   bool _checkActionHandlers() {
     // If there are no handlers specified, then we don't need to create new handlers
-    if (widget.onCustomMessageAction == null) {
+    if (widget.onCustomMessageAction == null && widget.onCustomConversationAction == null) {
```

Comment:
I'd split this function into `_checkMessageActionHandlers` and `_checkConversationHandlers` instead of 1 function that does 2 things.

Reply:
Done